.travis.yml

```diff
@@ -33,7 +33,7 @@ before_script:  # Coverage instructions - setup
   - export COVERAGE_PROCESS_START=$(pwd)/.coveragerc
 
 script:
-  - cd tests/epyccel/recognised_functions
+  - cd tests
```

Comment:
travis is no longer used for testing. To see your problem faster I suggest modifying [this line](https://github.com/pyccel/pyccel/blob/master/.github/workflows/Github_pytest.yml#L59)
to:
```
python -m pytest -rx -m "not parallel and c" epyccel/recognised_functions -k test_numpy_complex -x
```